grpc/tests/src/test/regression/ee_598.rs

```diff
@@ -18,7 +18,7 @@ const ARG_PUBLIC_KEY: &str = "public_key";
 const ARG_ENTRY_POINT: &str = "entry_point";
 const ARG_ACCOUNT_HASH: &str = "account_hash";
 
-const CONTRACT_MINT_BONDING: &str = "mint_bonding.wasm";
+const CONTRACT_AUCTION_BONDING: &str = "auction_bonding.wasm";
```

Comment:
wouldn't this be auction_bidding.wasm now?

Reply:
Good point - we should be changing file/crate names here as well.

Reply:
Fixed. While searching for `bond`/`unbond` I also got rid of a one regression test that was testing against `pos_bonding_purse` which doesn't make sense under auction.